Corona-Warn-App/src/main/java/de/rki/coronawarnapp/risk/RiskLevels.kt

```diff
@@ -0,0 +1,31 @@
+package de.rki.coronawarnapp.risk
+
+import com.google.android.gms.nearby.exposurenotification.ExposureSummary
+import de.rki.coronawarnapp.server.protocols.internal.AttenuationDurationOuterClass
+
+interface RiskLevels {
+
+    fun calculationNotPossibleBecauseNoKeys(): Boolean
+
+    fun calculationNotPossibleBecauseOfOutdatedResults(): Boolean
+
+    /**
+     * true if threshold is reached / if the duration of the activated tracing time is above the
+     * defined value
+     */
+    fun isActiveTracingTimeAboveThreshold(): Boolean
+
+    suspend fun isIncreasedRisk(): Boolean
+
+    suspend fun calculationNotPossibleBecauseTracingIsOff(): Boolean
```

Comment:
Moving things like early abort checks (e.g. `calculationNotPossibleBecauseTracingIsOff`) out of this interface? Into the task?

I think it's already A LOT better than before, but I have still trouble following the code path through the different steps. In my head I always try to imagine how the data is passed from one step to the next :thinking:.